scripts/xtensa-build-all.sh

```diff
@@ -8,7 +8,7 @@ set -e
 # Platforms with a toolchain available in the latest Docker image and
 # built by the -a option.
 DEFAULT_PLATFORMS=(  byt cht bdw hsw apl skl kbl cnl sue icl jsl \
-                    imx8 imx8x imx8m imx8ulp tgl tgl-h rn mt8195 )
+                    imx8 imx8x imx8m imx8ulp tgl tgl-h rn mt8186 mt8195 )
```

Comment:
I don't think `xtensa-mt8186-elf` is available yet, is it? If it is available then please add `mt8186` to `.github/workflows/pull-request.yml`. If not yet then please move `mt8186` to `SUPPORTED_PLATFORMS` for now.

Reply:
Hi marc, I check the current .github/workflows/pull-request.yml. there is no mt8195 strategy. should we update also ?